scripts/manage-imports.py

```diff
@@ -26,18 +26,16 @@ def ol_import_request(item, retries=5, servername=None, require_marc=True):
     """Requests OL to import an item and retries on server errors.
     """
     # logger uses batch_id:id for item.data identifier if no item.ia_id
-    _id = item.ia_id or "%s:%s" % (item.batch_id, item.id)
-    logger.info("importing %s", _id)
+    logger.info("importing %s", item.ia_id)
     for i in range(retries):
         if i != 0:
             logger.info("sleeping for 5 seconds before next attempt.")
             time.sleep(5)
         try:
             ol = get_ol(servername=servername)
-            if item.ia_id:
-                return ol.import_ocaid(item.ia_id, require_marc=require_marc)
-            else:
+            if item.data:
                 return ol.import_data(item.data)
+            return ol.import_ocaid(item.ia_id, require_marc=require_marc)
```

Comment:
## End Here!

Cron scripts use `add_items` to add entries to the import queue -- all it's really doing is populating the `import_item` postgres table.

The `import_all` method of `scripts/manage-imports.py` is what listens to and polls the database every minute or so to select ~1k pending db records for import.

Here, because now **all** records will have `ia_id`, we switch the ordering of the conditional logic such that the items having `data` is what triggers a data/json-based import. As opposed to before, the existence of the `ia_id` uniquely told us to import data from archive.org for the entry.